tips.json

```diff
@@ -254,6 +254,9 @@
 }, {
     "title": "Make git case sensitive.",
     "tip": "git config --global core.ignorecase false"
+},{
+    "title": "Add custom editors.",
+    "tip": "git config core.editor "MY_EDITOR""
```

Comment:
This should be `"tip": "git config core.editor '$EDITOR'"` basically it must be a valid JSON.